src/libraries/System.Private.Uri/src/System/Uri.cs

```diff
@@ -369,7 +369,7 @@ public Uri(string uriString)
         //
         //  Uri constructor. Assumes that input string is canonically escaped
         //
-        [Obsolete("The constructor has been deprecated. Please use new Uri(string). The dontEscape parameter is deprecated and is always false. https://go.microsoft.com/fwlink/?linkid=14202")]
+        [Obsolete("This constructor has been deprecated; the dontEscape parameter is always false. Use Uri(string) instead.")]
```

Comment:
> the dontEscape parameter is always false

This does not seem to match what the code does. As far as I can tell, `dontEscape` argument has some affect.

Reply:
It looks like you're right. I'll file a breakout issue for that instead of trying to resolve it here.

Reply:
#56840